src/indra_cogex/apps/home/__init__.py

```diff
@@ -50,4 +51,31 @@ def home():
         edge_counter=edge_counter,
         edge_labels=edge_labels,
         blueprints=current_app.blueprints,
+        pusher_app_key=pusher_key,
     )
+
+
+@home_blueprint.route("/new/guest", methods=["POST"])
```

Comment:
I think this would be better in its own blueprint, especially to make it easier to make a single page deploy of it

Reply:
I'm working on that in the follow-up branch :slightly_smiling_face: